### PR TITLE
fix using env variables

### PR DESCRIPTION
**Pull request by KorhanOzturk90**

### CODE REVIEW

File: conf/application-json-logger.xml [hunk 1]
```diff
@@ -5,9 +5,10 @@
         <encoder class="uk.gov.hmrc.play.logging.JsonEncoder"/>
     </appender>
 
-    <logger name="uk.gov" level="DEBUG"/>
+    <logger name="uk.gov" level="${logger.uk.gov:-WARN}"/>
```
Comment (konradwudkowski) [inline conf/application-json-logger.xml]: co znaczy WARN tutaj?

Comment (KorhanOzturk90) [inline conf/application-json-logger.xml]: myślę, że domyślnie WARN

Comment (konradwudkowski) [inline conf/application-json-logger.xml]: we log at INFO level, will it log with these settings?

Comment (KorhanOzturk90) [inline conf/application-json-logger.xml]: I'll update it to default to INFO in that case

**Pull request merged**